### DISCUSSION TITLE
How to plot material data in Ascent

### DISCUSSION THREAD

**Discussion by benjaminjeliot** [Alpine-DAV/ascent#1113] [Q&A] [answered]
Hi everyone,

How do I plot material data with Ascent?

I have instrumented a simulation app with Ascent and created some fields and a matset, however I can't figure out how to create a plot of the matset. I have successfully generated some plots of the mesh, and pseudocolor plots of the various field variables.

The material data is passed through to the Extracts Replay pipeline and  output as HDF5/JSON, so I can confirm that the material data is being set by loading the data in VisIt.

So far, the closest I can get to a material plot is to create additional field-type variable for the material ID and then use the pseudocolor plot. 

Is there a way to plot matsets with Ascent?

Many thanks,
Jim

**Comment by cyrush** [comment, marked as answer]
@benjaminjeliot  We don't have yet have a material interface reconstruction plot (like VisIt's Fill Boundary Blot) available Ascent. 

VTK-m does have support for material interface reconstruction, but we haven't wired anything up in Ascent yet.

**Reply by benjaminjeliot** [reply]
Thanks @cyrush.

That's useful to know -- it nice to a definitive answer.

Cheers,
Jim
